libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -52,10 +52,16 @@ + (NSString *)localizedString:(NSString *)localizationKey
 
 + (UIImage *)imageNamed:(NSString *)name
 {
-    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    //Get image from main bundle (this is to allow extracting SDK Assets when using template app as we include SDK Assets as part of the tempalte app)
```

Comment:
Yep, the issue is with how an asset catalog gets packaged in a pod. I am
trying to see if instead of an asset catalog I can use a folder containing
.png files of the images. Thoughts?

Sent from my iPhone

On Feb 8, 2016, at 1:37 PM, Kevin Hawkins <notifications@github.com> wrote:

In
libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m
<https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1379#discussion_r52223221>
:

>      NSAssert(name != nil, @"name must contain a value.");
> +    //Get image from main bundle (this is to allow extracting SDK Assets when using template app as we include SDK Assets as part of the tempalte app)

Hmmm. This seems like a bigger problem than the template apps, then. If
it's an issue there, it's an issue with apps that consume the SDK directly
through CocoaPods. We're going to need to find a more general solution to
supporting these image assets through CocoaPods.

—
Reply to this email directly or view it on GitHub
<https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1379/files#r52223221>
.